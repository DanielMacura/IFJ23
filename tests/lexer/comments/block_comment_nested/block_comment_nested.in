/*Beginning of test*/

let /*This is an inline block comment*/ x = 3.14    /*This is another inline block comment*/

/*
    /*
        This is a nested block comment  /* More nesting */  //This is an nested inline comment  /* More nesting */
    */
*/
