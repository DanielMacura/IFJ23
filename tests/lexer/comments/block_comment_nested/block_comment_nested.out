EOL
EOL
KEYWORD
IDENTIFIER
EQUALS
FLOAT:3.14
EOL
EOL
EOL
EOF
